requirements/pytorch/strategies.txt

```diff
@@ -1,6 +1,7 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
+colossalai==0.1.10
```

Comment:
I highly doubt this is the only possible version, we don't want to pin package version to the particular version

Reply:
yeah, i'll address this later once GPU ci is green

Reply:
but it is the only possible version, since versions older than 0.1.10 have different apis and 0.1.10 is the newest version. we can just set `colossalai>=0.1.10`.

Reply:
Yes, `colossalai>=0.1.10` is absolutely fine with me, but hard pin to a particular version is just weird.